centaur/src/it/scala/centaur/AbstractCentaurTestCaseSpec.scala

```diff
@@ -190,7 +190,7 @@ abstract class AbstractCentaurTestCaseSpec(cromwellBackends: List[String], cromw
     runTestIo.redeemWith(
       {
         case centaurTestException: CentaurTestException => maybeRetry(centaurTestException)
-        case _ => runTestIo
+        case nonCentaurException => IO.raiseError(nonCentaurException)
```

Comment:
Alternatively perhaps we could start by just logging the exception and then run the test action as we do now?

Reply:
Yeah, probably. This is not a final version of the PR yet. I'd like to see first what's actually going on under the hood there.